desktop/cmp/input/Select.js

```diff
@@ -436,7 +436,8 @@ export class Select extends HoistInput {
 
                 // But only return the matching options back to the combo.
                 return matchOpts;
-            });
+            })
+            .catch(e => console.error(e));
```

Comment:
john -- minor thing, but should we generally re-throw here as a matter of practice?

Reply:
Good call, yes I think we should do that, even though it seems to get swallowed by higher level code in this case